data/jibauni/jibauni 0 kanlanmo transport.txt

```diff
@@ -0,0 +1,628 @@
+# Copyright (c) 2024 by Loymdayddaud
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Jibauni: Kanlanmo 0"
+	name "Take Kanlanmo to <planet>"
+	description "Kanlanmo, who is a Jibauni in the shape of a Tyrannosaurus Rex, has asked to go to <planet> on your ship. He will explain more once you get there."
+	landing
+	source "Baitoon"
+	destination "Taltahard"
+	on offer
+		conversation
+			`Your landing location is a large set of old landing pads in the middle of a city, as there doesn't seem to be anywhere else available for spaceships to arrive. The city is filled with dinosaurs in the shape of the mythical Tyrannosaurus Rex, although here smaller then in old Earth fables. Soon after you land, one of these dinosaurs, wearing a dark green cloak, approaches your ship.`
+			choice
+				`	(Go out and meet it.)`
+					goto meeting
+				`	(Leave now, before something bad happens.)`
+					depart
+				`	(Stay in <ship>.)`
+			`	The dinosaur opens its mouth, and a moment later a growling sound rumbles through the air. Then you hear the words, "Could you exit your ship to consult with me?"`
+				to display
+					not "language: Coalition"
+				goto initialchoice1
+			action
+				set "jibauni knowledge: the Jibauni have Heliarch translators"
+			`	The dinosaur opens its mouth to say something, and you hear a growling sound. Then you notice that it has a Coalition translator box around its neck. Before you can do anything the box says, "Could you exit your ship to consult with me?"`
```

Comment:
`	The dinosaur-like alien's terrifying maw opens wide and unleashes a deep growling sound that causes your very bones to vibrate. Although it's difficult to pry your eyes from the rows of fearsome carnivorous teeth on display, you're drawn to light reflecting off a metallic bauble danging from the alien's neck. To your surprise, you recognize the Coalition translator device; it whirs to life and a squeaky artificial voice says, "Could you exit your ship to consult with me?"`